### PR TITLE
Display owning user on external services list

### PR DESCRIPTION
**Pull request by eseliger**
List:
image
Detail:
image

Closes https://github.com/sourcegraph/sourcegraph/issues/14976

### CODE REVIEW

**Review by artemruts — APPROVED** [review]
FE part looks good to me, thank you Erik!

**Comment by artemruts** [comment]
@arussellsaw can you confirm that sync time logic is still true? I didn't check the GO part.

**Review by ryanslade — APPROVED** [review]
Nice!

File: cmd/frontend/graphqlbackend/external_services_test.go [hunk 1]
```diff
@@ -241,7 +241,7 @@ func TestAddExternalService(t *testing.T) {
 					kind
 					displayName
 					config
-					namespace
+					namespace { id }
```
Comment (unknwon) [inline cmd/frontend/graphqlbackend/external_services_test.go:244]: FYI this is a breaking change, but I don't think it's worth blocking us.

**Review by unknwon — APPROVED** [review]
Thanks!

**Review by arussellsaw — APPROVED** [review]

**Pull request merged**